dataland-frontend/tests/e2e/fixtures/fakeFixtures.ts

```diff
@@ -1,25 +1,43 @@
-const { faker } = require('@faker-js/faker');
+const {faker} = require('@faker-js/faker');
+const backend = require( "../../../build/clients/backend/backendOpenApi.json")
 const fs = require('fs')
 // sets locale to de
 faker.locale = 'de';
 
 function generateCompanies() {
     let companies = []
 
-    for (let id=1; id <= 100; id++) {
+    for (let id = 1; id <= 100; id++) {
         let companyName = faker.company.companyName();
         let headquarters = faker.address.city();
         let sector = faker.company.bsNoun();
         let marketCap = faker.mersenne.rand(50000, 10000000);
         let reportingDateOfMarketCap = faker.date.past().toISOString().split('T')[0]
+        let indices = faker.random.arrayElements( backend.components.schemas.CompanyInformation.properties["indices"].items.enum );
+        let identifiers = faker.random.arrayElements([
+            {
+                "identifierType": backend.components.schemas.CompanyIdentifier.properties.identifierType.enum[0],
+                "identifierValue": "529900W18LQJJN6SJ336"
+            },
+            {
+                "identifierType": backend.components.schemas.CompanyIdentifier.properties.identifierType.enum[1],
+                "identifierValue": "529900W18LQJJN6SJ336"
+            },
+            {
+                "identifierType": backend.components.schemas.CompanyIdentifier.properties.identifierType.enum[2],
+                "identifierValue": "529900W18LQJJN6SJ336"
```

Comment:
Ich würde vorschlagen (z.B. über einen concat mit einem festen string o.Ä.), dass wir dafür sorgen, dass `identifierValues` von verschiedenen `identifierTypes` niemals exakt denselben Wert haben sollten. Und von verschiedenen companies erst recht nicht.

Ich erinnere mich nämlich an eine Pair-Programming-Session mit Siegfried letzte Woche, in der uns aufgefallen war, dass sonst bestimmte Tests failen könnten.

Unabhängig davon spiegelt es auch die Realität besser wieder. (Oder?)